code/public/index.html

```diff
@@ -4,31 +4,31 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
 
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>Technigo React App</title>
+  <!-- OG Tags -->
+  <meta property="og:title" content="Linnea's Todo List" />
+  <meta property="og:description" content="A Todo list made by Linnea Frisk"/>
+  <meta property="og:image" content="https://i.postimg.cc/4dRCj3yV/pexels-eugene-liashchevskyi-10843173.jpg"/>
+  <meta property="og:url" content="https://project-todo-linneafrisk.netlify.app/"/>
+
+    <!-- Google Fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400&family=Staatliches&display=swap" rel="stylesheet">
+
+    <!-- Favicon -->
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
```

Comment:
I'm curious - why adding several sizes for favicons? Is that for different devices such as tablets etc?